code/game/objects/items/devices/last_shelter.dm

```diff
@@ -115,3 +115,15 @@ GLOBAL_DATUM(last_shelter, /obj/item/device/last_shelter)
 	sleep(request_timeout)
 	agree_time_out = TRUE
 	return MN
+
+// OCCULUS EDIT: MAKING THE LAST SHELTER WORK
+
+/mob/living/lost_in_space()
+	var/obj/item/weapon/implant/core_implant/cruciform/CI = get_core_implant(/obj/item/weapon/implant/core_implant/cruciform)
+	if(CI)
+		gib()
```

Comment:
Can you throw a popup here if the target is consious? Some people may not want to be recovered.